internal/scorecard/plugins/config.go

```diff
@@ -24,7 +24,6 @@ import (
 
 const (
 	NamespaceOpt          = "namespace"
-	KubeconfigOpt         = "kubeconfig"
 	InitTimeoutOpt        = "init-timeout"
 	OlmDeployedOpt        = "olm-deployed"
 	CSVPathOpt            = "csv-path"
```

Comment:
Are any of these consts used anymore? If not, let's remove them or move into the `cmd` tree.

Reply:
removed.